spec/acceptance/basic_posix_acl_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+describe 'posix_acl' do
+  context 'with default behaviour' do
+    it_behaves_like 'an idempotent resource' do
+      pp = <<-EOS
+      user { 'blub':}
+      file { '/opt/test':
+        ensure => directory,
+        owner  => root,
+        group  => root,
+        mode   => '2770',
+      }
+      -> posix_acl { '/opt/test':
+        action     => exact,
+        permission => [
+          'user:blub:r-x'
+        ],
+        provider   => posixacl,
+        recurse    => false,
+      }
+      EOS
+      let(:manifest) do
+        pp
```

Comment:
Why the separate variable instead of just putting it inline here?

Reply:
fixed it